index.html

```diff
@@ -348,52 +348,18 @@ <h1 class="head3">Skills</h1>
           </p>
         </div>
         <div class="form-contents">
-          <form
-            class="text-form"
-            action="https://formspree.io/f/xbjwvddv"
-            method="POST"
-          >
-            <input
-              class="fullname-input"
-              type="text"
-              name="name"
-              placeholder="Full Name"
-              maxlength="30"
-              required
-            />
-            <input
-              class="firstname-input"
-              type="text"
-              name="firstname"
-              placeholder="First Name"
-              maxlength="30"
-              required
-            />
-            <input
-              class="lastname-input"
-              type="text"
-              name="lastname"
-              placeholder="Last Name"
-              maxlength="30"
-              required
-            />
-            <input
-              type="email"
-              name="email"
-              placeholder="Email Address"
-              required
-            />
-            <textarea
-              cols="30"
-              rows="10"
-              name="message"
-              maxlength="500"
-              placeholder="Enter text here"
-            ></textarea>
+          <form class="text-form" action="https://formspree.io/f/xbjwvddv" method="POST">
+            <input class="fullname-input" type="text" name="name" placeholder="Full Name" maxlength="30" required/>
+            <!-- <input class="firstname-input" type="text" name="firstname" placeholder="First Name" maxlength="30" required/>
+            <input class="lastname-input" type="text" name="lastname" placeholder="Last Name" maxlength="30" required/> -->
```

Comment:
Great job team 🤝

- [ ] [Optional] You may kindly remove all commented code from your HTML file as this is regarded as a bad practice.